<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="diaryText">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>471</width>
      <height>401</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="diaryList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="diaryListLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>일기 목록</string>
    </property>
   </widget>
   <widget class="QLabel" name="fromLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>보낸 사람: </string>
    </property>
   </widget>
   <widget class="QLabel" name="toLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>받는 사람: </string>
    </property>
   </widget>
   <widget class="QLabel" name="subjectLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>제목:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fromEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="toEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="subjectEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="propertyButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>이메일 자세하게 보기...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="attatchmentGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>399</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>첨부파일</string>
    </property>
    <widget class="QLabel" name="attatchmentLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>62</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>파일 선택:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="attatchmentComboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="attatchmentSaveAsButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>231</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>첨부파일 저장...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuTestMenu">
    <property name="title">
     <string>테스트</string>
    </property>
    <addaction name="actionConsoleAuthentication"/>
    <addaction name="actionBrowserAuthentication"/>
    <addaction name="separator"/>
    <addaction name="actionList_Mailboxes"/>
    <addaction name="actionBuild_Local_Structure"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTestMenu"/>
  </widget>
  <action name="actionOptions">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionConsoleAuthentication">
   <property name="text">
    <string>콘솔에서 인증하기...</string>
   </property>
  </action>
  <action name="actionList_Mailboxes">
   <property name="text">
    <string>List Mailboxes</string>
   </property>
  </action>
  <action name="actionBuild_Local_Structure">
   <property name="text">
    <string>Build Local Structure</string>
   </property>
  </action>
  <action name="actionBrowserAuthentication">
   <property name="text">
    <string>웹브라우저로 인증하기...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
